content/en/blog/2025/12/locomo-results-v0.2/index.md

```diff
@@ -0,0 +1,200 @@
+---
+title: "MemMachine v0.2 release - Locomo benchmark results"
+date: 2025-12-01T13:22:00-08:00
+featured_image: "featured_image.png"
+tags: ["AI Agent", "LoCoMo Benchmark", "Generative AI", "LLM", "Agent Memory", "featured"]
+author: "The MemMachine Team"
+description: "This paper presents the results of the Mem0 evaluation of Locomo benchmark for the v0.2 release of memMachine."
+aliases:
+  - /blog/2025/12/locomo-results-v0.2/
+---
+
+# MemMachine v0.2 release - Locomo benchmark results
```

Comment:
Remove this header (h1). The post automatically creates it from the `title:` metadata field. This will cause duplicate headers to be rendered.